lessons/Algebra2/exploring-exponential-models/langs/en-us/index.adoc

```diff
@@ -0,0 +1,200 @@
+
+[.lesson-instruction]
+- Complete the *second* section on @printable-exercise{graphing-models.adoc}, using the first screen in @starter-file{alg2-covid-desmos} (Quadratic Models).
+
+Quadratic models capture _parabolic relationships_, where one quantity varies based on the square of another. In quadratic models, we expect the response variable to grow by differing amounts over equal intervals in the explanatory variable.
+
+[.lesson-instruction]
+- Write the best quadratic model you can find at the bottom of @printable-exercise{exploring-covid.adoc}.
+- Class discussion: Are quadratic models a good fit for this data? Why or why not?
+
+Have students share their resulting models. Which one fits best?
+
+@right{@image{images/MA-covid-quadratic.png, 300}} Quadratic models change their rate of growth over time, which definitely makes them a better fit for this data than linear ones. But they still don't have the explosive growth we need to model this data, which starts out incredibly slow and then suddenly takes off like a rocket.
+
+=== Synthesize
+
+- Do you think the data for MA shows a linear relationship? Why or why not?
+- Do you think this data shows a quadratic relationship? Why or why not?
+- Do you think this data shows some other kind of relationship? Why or why not?
+
+== Fitting Exponential Models @duration{20 minutes}
+
+=== Overview
+
+Students are introduced to exponential models, and extend their sampling techniques to exponential relationships. Students continue experimenting in Desmos, but eventually switch back to Pyret to formalize their understanding.
+
+=== Launch
+
+There is, however, a class of functions that grow even faster than quadratics: *exponential functions*.
+
+[.lesson-point]
+Linear functions grow by equal *amounts* over equal intervals (adding _n_ each time). Exponential functions grow by equal *factors* over equal intervals (multiplying by _n_ each time).
+
+Exponential relationships show up all the time!
+
+- Cells that constantly divide, doubling the total number of cells each time
+- A tree that's on fire will likely catch neighboring trees on fire, so that the fire spreads to 2 or more trees each time
+- Money in a savings account grows by a certain percentage each year, meaning every year there's that much more money to grow
+
+
+[.lesson-instruction]
+Can you think of other real-world examples where a relationship is likely exponential?
+
+Exponential models have the form @math{y = ab^x + k}
+
+- @math{k} is the vertical shift, which is the amount the graph is shifted up or down the y-axis. @math{k} contributes the same amount to the value of the function for all values of @math{x}
```

Comment:
Why not have them make a guess about this before telling them... it looks a lot like y-intercepts in linear functions...  or have explore what k, a and b do in Desmos (like they did with quadratics) before telling them.

Reply:
Oooh, great idea! 

I've moved the first part of the exponential-models workbook page into the launch section, so that students will have some time to play with the terms _before_ discussing what they contribute. This has the same effect of adding your desmos link.